### ISSUE TITLE
Possible fix for the left facing sprite

### ISSUE THREAD

**Issue by EzTargetUK** [benheck/gameBadge#1]
In gameBadge.c the following lines need changing to fix the issue with the left facing sprite.

if (mirror == -1) {										//If mirror draw sprites from right to left
	bitmap += xSize - 1;
}

and 

if (mirror == -1) {														//Since mirrored = going backwards in memory we have to skip 2 row widths to get to the right-hand start of the next one
	bitmap += (xSize * 2);
}

At least this is what worked on the Pico that I'm using instead of the MCP that your using.

**Comment by benheck** [comment]
Yup, found that after making the video but I'll double check you fix
against mine.

Thanks!
